cmake/options.cmake

```diff
@@ -159,6 +159,14 @@ set(ONEDNN_ENABLE_GEMM_KERNELS_ISA "ALL" CACHE STRING
       SSE41 < AVX2 < AVX512 < AMX (or ALL). It means that if user selects, e.g.
       AVX2 ISA, SSE41 kernels will also present at build time.")
 
+set(DNNL_HIP_TARGET_ARCH "" CACHE STRING
+    "Specifies the target offload architecture. Required when compiling
+    sycl kernels for AMD target backend.")
+if("${DNNL_HIP_TARGET_ARCH}" STREQUAL "")
+    add_definitions(-DDNNL_HIP_DISABLE_SYCL_KERNELS=1)
+    set(DNNL_HIP_DISABLE_SYCL_KERNELS TRUE)
```

Comment:
After adjusting name of `DNNL_HIP_TARGET_ARCH` please adjust these accordingly.

Reply:
Done.